lib/rspec/rails/configuration.rb

```diff
@@ -157,6 +163,18 @@ def filter_rails_from_backtrace!
           filter_gems_from_backtrace "activemodel", "activerecord",
                                      "activesupport", "activejob"
         end
+
+        # @deprecated TestFixtures#fixture_path is deprecated and will be removed in Rails 7.2
+        if ::Rails::VERSION::STRING >= "7.1.0"
+          def fixture_path=(path)
+            RSpec.deprecate(
+              "config.fixture_path =  #{path.inspect}",
+              replacement: "config.fixture_paths = [#{path.inspect}]",
+              message: "Rails 7.1 has deprecated the singular fixture_path in favour of an array"
```

Comment:
And we should add.
```
Rails 7.1 has deprecated the singular fixture_path. You should migrate to plural fixture_paths and wrap value into an array.

config.fixture_paths = ["#{Rails.root}/spec/fixtures/"]

```

Reply:
`config.fixture_paths = ["#{Rails.root}/spec/fixtures/"]` is implied by the replacement text, would support a message tweak to [or similar]:

`Rails 7.1 has deprecated the singular fixture_path. You should migrate to plural:`

Reply:
The singular could also error if given an array

Reply:
Do you want a PR @JonRowe ?

Reply:
Always :p

Reply:
https://github.com/rspec/rspec-rails/pull/2720